File: 백준/Bronze/10250. ACM 호텔/ACM 호텔.py
T = int(input())
for i in range(T):
    H,W,N = map(int,input().split(" "))
    floor=0
    room=0
    if(1<=H and W<=99 and 1<=N and N<=H*W):
            if(N%H == 0):
                floor = H
                room = N//H
            else:
                floor=N%H
                room=N//H+1
            print(floor*100+room)

File: 백준/Bronze/10250. ACM 호텔/ACM 호텔.c
#include<stdio.h>
int main() {
	int T, H, W, N,floor,room,no;
	scanf("%d",&T);
	for (int i = 0; i < T; i++) {
		scanf("%d %d %d", &H, &W, &N);
		if (1 <= H && W <= 99 && 1 <= N && N <= H * W){
			floor = 0;
			room = 0;
			if (N % H == 0) {
				floor = H;
				room = N / H;
			}
			else {
				floor = N % H;
				room = N / H + 1;
			}
			no = floor * 100 + room;
			printf("%d\n", no);
		}
	}
	return 0;
}